docs/dietpi_tools/system_maintenance.md

```diff
@@ -67,53 +67,69 @@ dietpi-backup
+    The file containing the filter definitions is `/boot/dietpi/.dietpi-backup_inc_exc`.
 
-```sh
-dietpi-cron
-```
+=== "Space check"
 
-Please keep in mind that all other daily `cron` based procedures are also started at this changed time.
+    A space check on the target location prior to the backup process can be enabled/disabled. This might be an option if there is surely enough disk space available.  
+    Enabling the space check makes the backup a bit more safe, disabling it speeds it up.
 
-### Backup file selection (Filter)
+=== "Logging"
 
-The definition which files are used for the backup procedure is defined via the option "Filter" (see screenshot above). This opens `nano` to edit the include/exclude definitions for the backup.  
-The filter definition syntax is described within the file itself.
+    Logging information about the backup procedure is given within the files `.dietpi-backup_stats` and `.dietpi-backup.log` which are located in the backup target directory ("Location" option):
 
-![DietPi-Backup filter option screenshot](../assets/images/dietpi-backup_filter-option.jpg){: width="681" height="330" loading="lazy"}
+    - `.dietpi-backup_stats` gives a list of completed operations with time and date
+    - `.dietpi-backup.log` gives a list of every processed file
 
-The file containing the filter definitions is `/boot/dietpi/.dietpi-backup_inc_exc`.
+### Location of the backup
 
-### Space check
+The location of the backup can be set to any directory which is not part of the backup itself. Especially, these options are typical:
 
-A space check on the target location prior to the backup process can be enabled/disabled. This might be an option if there is surely enough disk space available.  
-Enabling the space check makes the backup a bit more safe, disabling it speeds it up.
+- Backup to/Restore from a connected USB stick or USB disk
+- Backup to/Restore from a large system disk mounted below `/media` (because `/media` is not backed up by default)
+- Backup to/Restore from an network drive (Linux file system needed!)
 
-### Logging
+!!! info "Backup to/Restore from a network drive: Only Linux file systems are usable (NFS)"
+
+    DietPi-Backup is purely based on `Rsync`, i.e., DietPi-Backup uses a backup on a file by file basis and is not based on an archive algorithm. This leads to the fact that file permissions/attributes are only preserved correctly in case of a Linux file system as the target directory. In other cases (e.g. a Samba share), file permissions are not saved in the right manner and DietPi-Backup will not work appropriate.
+
+    **Therefore, DietPi-Backup should only be used to/from a directory which handles Linux file attributes like NFS.**
+
+    Remark: In the case that the `rsync` package is not installed, this is done automatically once you start a backup or restore.
```

Comment:
```suggestion
```
Probably no need to mention this, user will see it, and there is no harm if one installs it manually.